test/socket.io.js

```diff
@@ -1511,7 +1511,7 @@ describe('socket.io', function(){
     it('should handle very large json', function(done){
       this.timeout(30000);
       var srv = http();
-      var sio = io(srv);
+      var sio = io(srv, { perMessageDeflate: false });
```

Comment:
Hi! Well, I think it causes some builds (like this one for example :angel:  https://travis-ci.org/socketio/socket.io/jobs/92141401) to pathetically fail (by timeout) due to the perMessageDeflate feature (which is somehow great, don't get me wrong!).